src/SprykerEco/Zed/Unzer/Persistence/Propel/Schema/spy_unzer.schema.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\Unzer\Persistence" package="src.Orm.Zed.Unzer.Persistence">
+
+    <table name="spy_payment_unzer" idMethod="native" allowPkInsert="true" isCrossRef="true">
```

Comment:
Pay attention that:
Table name has to contain a Module name after the prefix:
- `spy_[module]`
- `spy_[module]_[suffix]`

E.g. `spy_customer` or `spy_customer_address` or `spy_customer_address_book`